casr/tests/tests.rs

```diff
@@ -5416,6 +5416,20 @@ fn test_casr_libfuzzer_jazzer_js_xml2js() {
         .status()
         .expect("failed to unzip xml2js.zip");
 
+    let Ok(npm_path) = which::which("npm") else {
+        panic!("No npm is found.");
+    };
+
+    let npm = Command::new("bash")
+        .current_dir(&test_dir)
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .arg("-c")
+        .arg(format!("{} install xml2js", &npm_path.display()))
```

Comment:
Are we going to always install npm package in user home directory just for tests? Can we run it from within the test directory?

Reply:
Maybe, we can just check that package exists and install it in CI?

Reply:
This package is installed in test directory, line 5424